index.d.ts

```diff
@@ -66,7 +66,7 @@ declare namespace Eris {
   type GuildIntegrationExpireBehavior = Constants["GuildIntegrationExpireBehavior"][keyof Constants["GuildIntegrationExpireBehavior"]];
 
   // Message
-  type ActionRowComponents = Button | SelectMenu;
+  type ActionRowComponents = Button | SelectMenu | TextInput;
```

Comment:
Is this the best way to do it? as Text inputs are exclusive to modals?

Reply:
Text input is a component type according to the docs so...?